main.html

```diff
@@ -1,77 +1,233 @@
-    l = sin(theta)
-    F = cos(2 * pi * bl_length * l / wavelength)
-    F2 = cos(2 * pi * 2 * bl_length * l / wavelength)
-
-    plt.plot(l, F, c='#cc0000', label=&quot;Baseline 1-2&quot;)
-    plt.plot(l, F2, c='#0000cc', label=&quot;Baseline 1-3&quot;)
-    plt.xlabel(&quot;$sin(\\theta)$&quot;)
-    plt.ylabel(&quot;Fringe amplitude&quot;)
-    plt.ylim(-2, 2)
-    plt.legend()
-
-f = interact(plot_fringe, bl_length=(1, 100), wavelength=(1, 100))
-&lt;/pre&gt;
-</pre>
+<h2>Introduction</h2>
+
+<p>
+  InnerSource is the strategy of applying open source practices and principles to internal software development.
+  This strategy results in shared software that is responsive and nimble to the changing needs of its many internal consumers.
+</p>
+
+<p>
+  This Oriole is an introduction to InnerSource.
+  It teaches how to recognize situations that are good candidates for InnerSource.
+  We'll outline at a high level how InnerSource can help in those situations.
+  You'll become familiar with key terms used when discussing InnerSource.
+  We'll also enumerate the key principles upon-which InnerSource is based and the benefits seen when it is applied effectively.
+</p>
+
+<h2>A Common Scenario</h2>
+
+<p>
+  Imagine two teams delivering separate pieces of software with one team's software depending on that of the other.
+  An example could be an API service that depends on a shared library for part of its functionality.
+  This type of situation is common in a large enterprise where a single team producing software may have dozens or hundreds of consumers.
+  When consuming teams need many features, producing teams normally have some sort of requirements and prioritization process to decide which features they will work on.
+  For feature requests that are not prioritized, the consuming team is faced with 3 choices, each of which come with their own drawbacks.
+</p>
+
+<ol>
+  <li>
+    <p>
+      <strong>Wait it out</strong>.
+      The consuming team may do nothing and limp along without the requested functionality.
+      This option requires the least amount of work on their side.
+      Depending on the benefit of the feature request, <em>Wait it out</em> might be just fine.
+      However, waiting may come with real amounts of pain, especially if the requested functionalty is <em>never</em> delivered.
+    </p>
+  </li>
+  <li>
+    <p>
+      <strong>Workaround</strong>.
+      A consuming team that doesn't want to wait may do extra work somewhere else to compensate for the absence of their requested feature.
+      This extra work may come as change in the consuming project.
+      Alternatively, the consuming team may create a new project that meets their needs and replaces their use of all or part of the producing team's system (code/project duplication).
+    </p>
+    <p>
+      This strategy allows the consuming team to get the requested feature via their own efforts only (no hard dependencies).
```

Comment:
Really minor nit here.  I don't know what a _hard_ dependency is.  Is this a _**direct**_ dependency?  That might be a better way to describe this aside since it is so short.  

Why not just include these parenthetical into the main idea?

Reply:
Updated.